schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/formatter/dcat/dcat-core-dataservice.xsl

```diff
@@ -4,24 +4,36 @@
                 xmlns:xs="http://www.w3.org/2001/XMLSchema"
                 xmlns:xlink="http://www.w3.org/1999/xlink"
                 xmlns:cit="http://standards.iso.org/iso/19115/-3/cit/2.0"
-                xmlns:srv="http://standards.iso.org/iso/19115/-3/srv/2.1"
+                xmlns:srv="http://standards.iso.org/iso/19115/-3/srv/2.0"
                 xmlns:gco="http://standards.iso.org/iso/19115/-3/gco/1.0"
+                xmlns:gcx="http://standards.iso.org/iso/19115/-3/gcx/1.0"
                 xmlns:dcat="http://www.w3.org/ns/dcat#"
                 xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
                 exclude-result-prefixes="#all">
 
-  <xsl:variable name="endpointDescriptionProtocols"
-                as="xs:string*"
-                select="('OpenAPI', 'GetCapabilities')"/>
+  <!--
+  Filter to define which service operation has to be considered
+  as an endpoint URL or description depending on linkage and protocol.
+  -->
+  <xsl:param name="endpointDescriptionUrllExpression"
```

Comment:
I'm not sure if instead `endpointDescriptionUrllExpression`  you wanted to use `endpointDescriptionUrlsExpression`?

In any case a minimal detail.

Reply:
Good catch. Fixed